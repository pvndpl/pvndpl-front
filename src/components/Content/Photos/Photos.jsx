import React from 'react';
import PropTypes from 'prop-types';
import styles from './Photos.module.css';
import HeaderContent from "../HeaderContent/HeaderContent";

const Photos = () => (
  <div className={styles.Photos} data-testid="Photos">
    <HeaderContent name={"Александр Великодный"} count={13} section={"Фотографии"}/>
      <div className={styles.album}>

      </div>
  </div>
);

export default Photos;
