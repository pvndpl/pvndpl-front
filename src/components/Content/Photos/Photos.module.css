.Photos {}

.albums{
    display: grid;
    grid-template-columns: repeat(5, 1fr);
    margin: 1em;
    border-radius: 12px;
    background-color: #fff;
    box-shadow: 0 0 40px 0 rgba(94, 92, 154, .06);
}

.photo{
    width: 266px;
    height: 266px;
    border-radius: 12px;
    margin: 1em;
    cursor: pointer;
}

@media screen and (max-width: 1162px) {
    .albums {
        grid-template-columns: repeat(3, auto);
    }
    .headerInfoImg {
        height: 128px;
        width: 128px;
    }

}

@media screen and (max-width: 916px) {
    .albums {
        grid-template-columns: repeat(2, auto);
    }

    .headerInfoImg {
        height: 128px;
        width: 128px;
    }

}

@media screen and (max-width: 630px) {
    .albums {
        grid-template-columns: 1fr;
    }

    .headerInfoImg {
        height: 128px;
        width: 128px;
    }

}