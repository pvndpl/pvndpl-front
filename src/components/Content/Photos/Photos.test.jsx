import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Photos from './Photos';

describe('<Photos />', () => {
  test('it should mount', () => {
    const element = render(<Photos />);

    expect(element).toMatchSnapshot();

  });
});