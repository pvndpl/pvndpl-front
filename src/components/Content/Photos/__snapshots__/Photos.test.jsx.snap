// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Photos /> it should mount 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Photos"
        data-testid="Photos"
      >
        <div
          class="HeaderContent"
          data-testid="HeaderContent"
        >
          <p
            class="name"
          >
            Александр Великодный
          </p>
          <h2
            class="section"
          >
            Фотографии
            <span
              class="count"
            >
              13
            </span>
          </h2>
        </div>
        <div
          class="albums"
        >
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
          <img
            class="photo"
            src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Photos"
      data-testid="Photos"
    >
      <div
        class="HeaderContent"
        data-testid="HeaderContent"
      >
        <p
          class="name"
        >
          Александр Великодный
        </p>
        <h2
          class="section"
        >
          Фотографии
          <span
            class="count"
          >
            13
          </span>
        </h2>
      </div>
      <div
        class="albums"
      >
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
        <img
          class="photo"
          src="https://sun9-20.userapi.com/impg/lcUChi9t4AE4CmbvwLOAaPDsnyIVqSD6-ldjGw/qYgMUAPs-n0.jpg?size=1280x963&quality=96&sign=93c1e800bd1f11098a15659a713ad0de&type=album"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
